.gitignore

```diff
@@ -5,4 +5,4 @@ coverage
 .DS_Store
 .idea
 .env
-config/meta.dev.json
```

Comment:
Should we leave the old location in addition so it doesn't get committed by accident?

Reply:
+1

Reply:
Yes absolutely, just a left over after multiple `rebase -i`!